src/vec_common_ppc.h

```diff
@@ -28,14 +28,173 @@
 /*!
  * \file  vec_common_ppc.h
  * \brief Common definitions and typedef used by the collection of
- * Power Vector Library headers.
- *
- *  \section includes
- *  - Typedefs as short names of common vector types.
- *  - Union used to transfer 128-bit data between vector and
- *  non-vector types.
- *  - Helper macros that make declaring constants and accessing
- *  elements, a little easier.
+ * Power Vector Library (pveclib) headers.
+ *
+ * This includes:
+ * - Typedefs as short names of common vector types.
+ * - Union used to transfer 128-bit data between vector and
+ * non-vector types.
+ * - Helper macros that make declaring constants and accessing
+ * elements, a little easier.
```

Comment:
[_nit_] You didn't change this, I know, but the comma should be removed here.

Reply:
OK